README.md

```diff
@@ -35,7 +35,8 @@ The simplest example of how to use the package is:
 ```
 from adapter.i_o import IO
 
-input_loader = IO(<fullpath_to_the_main_input_file>)
+input_loader = IO(<fullpath_to_the_main_input_file>, 
+                               os_mapping={'win32': 'C:', 'darwin': '/Volumes/A', 'linux': '/media/A'})

```

Comment:
This would be better as an additional example, with a note that one should customize the paths for their local machines/servers.

Reply:
Please use the text I provided in the issue description, and paste it after the existing example, rather than modifying the existing example. Thanks!

Reply:
My bad. I updated the example.